### ISSUE TITLE
Cultivated Tech Industrial Hemp plants spam seeds when right clicked

### ISSUE THREAD

**Issue by MongoTheElder** [Darkosto/SevTech-Ages#209]
The Cultivated Tech Industrial Hemp plants will give seeds at any growth stage when right clicked (leaving the plant intact).

![2017-11-23_09 43 36](https://user-images.githubusercontent.com/29290246/33178081-d5620dba-d032-11e7-9bbb-c14a05003239.png)

**Comment by Darkosto** [comment]
Hey @EwyBoy is this still a thing? :P

**Comment by EwyBoy** [comment]
This is new to me, but should be easy to fix. Give me 5 minutes and some red bull and I can have a build out for you fixing this.

**Comment by EwyBoy** [comment]
![24020075_1740858949258642_1592131404_n](https://user-images.githubusercontent.com/5883716/33195190-dfe0deda-d0d5-11e7-93c9-b73f7a938c11.jpg)

14 Red Bulls later and I got a fixed build for ya! https://minecraft.curseforge.com/projects/cultivated-tech/files/2502908

Thanks to this I was able to track down a couple other issues with the crops and soil. Thank you for the bug report @MongoTheElder and have a nice day!
